src/sdk/lsf-sdk.js

```diff
@@ -390,7 +390,12 @@ export class LSFWrapper {
           if (c) {
             c.history.freeze();
             console.log("Applying draft");
-            c.addVersions({ draft: draft.result });
+            const draftWithLeadTime = draft.result.map((d, i) => {
+              if (i === 0) return { ...d, leadTime: draft.lead_time };
```

Comment:
This would only get the first lead_time from results, is that expected? I was thinking this would be an accumulation of the results' lead_time.

Reply:
There is only one lead time saved in the drafts api and it is, I believe for a singular unsaved draft which I think is at the front of the list always. So, this should put it right where it should be. It is still unclear to me why there are a lot of drafts returned in the api but only one displayed. We could put the same lead time in all of the drafts but that doesn't seem correct. Another option is to save and send draft time separately from the drafts themselves.

Reply:
is there a null safe fallback for the lead_time just to be safe?

Reply:
never mind I do see that lsf-utils.ts handles that - even though `APIAnnotation` defines it as `number | null`

Reply:
lead_time in drafts saved on a top level like in annotations, this lead_time is different lead_time and should not be touched here